### PR TITLE
Adding an important note how to list all the available SOAP Web Services

### PR DESCRIPTION
**Pull request by eduard13**
## Purpose of this pull request

This pull request (PR) improves the SOAP API documentation, by providing the way of how all the available Web Services can be listed.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/get-started/soap/soap-web-api-calls.html
- https://devdocs.magento.com/guides/v2.2/get-started/soap/soap-web-api-calls.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/get-started/soap/soap-web-api-calls.md [hunk 1]
```diff
@@ -17,33 +17,10 @@ Each Magento service interface that is part of a [service contract](https://glos
...
-         <td>customer, catalogProduct</td>
-         <td>http://magentohost/soap/custom_store?wsdl&services=customerCustomerAccountServiceV1,catalogProductV2</td>
-         <td>\Magento\Customer\Service\V1\CustomerAccountServiceInterface, \Magento\Catalog\Service\V2\ProductInterface</td>
-      </tr>
-   </tbody>
-</table>
+| Service | WSDL endpoint URL | Available services |
+| --------- | ---------- | ------------------------------------------ |
+| customer | http://magentohost/soap?wsdl&services=customerV1 | \Magento\Customer\Service\V1\CustomerService |
+| customer, catalogProduct | http://magentohost/soap/custom_store?wsdl&services=customerCustomerAccountServiceV1,catalogProductV2 | \Magento\Customer\Service\V1\CustomerAccountServiceInterface, \Magento\Catalog\Service\V2\ProductInterface |
```
Comment (keharper) [inline guides/v2.2/get-started/soap/soap-web-api-calls.md]: Is this correct? Do we have any V2s?

File: guides/v2.2/get-started/soap/soap-web-api-calls.md [hunk 2]
```diff
@@ -52,6 +29,27 @@ The WSDL URL follows the following pattern:
 You must specify each service version in the endpoint URL.
 
 This way, you can have a strict contract between your application and the service provider.
+
+If you want an overview to all the available Web Services, just use the following url format to get a list of all SOAP Services:
```
Comment (keharper) [inline guides/v2.2/get-started/soap/soap-web-api-calls.md]:
```suggestion
If you want an overview to all the available Web Services, use the following URL format to get a list of all SOAP Services:
```

File: guides/v2.2/get-started/soap/soap-web-api-calls.md (hunk 1 above)
Comment (eduard13) [reply guides/v2.2/get-started/soap/soap-web-api-calls.md]: Looks like it's not. Currently there are only V1 services available. Also looks like the `Magento\Customer\Service\V1\CustomerAccountServiceInterface` and `Magento\Customer\Service\V1\CustomerAccountService` classes are obsolete, so they were also adjusted with the corresponding API classes.

Thanks.

File: guides/v2.2/get-started/soap/soap-web-api-calls.md (hunk 2 above)
Comment (eduard13) [reply guides/v2.2/get-started/soap/soap-web-api-calls.md]: Adjusted

**Comment by eduard13** [comment]
Hi @keharper, thank you for your reviews, could you please check the latest changes?

**Pull request merged**